### ISSUE TITLE
Virustotal alerts

### ISSUE THREAD

**Issue by dslgr** [ililim/dual-key-remap#49]
Hi,
I'm not sure what there is to do about it but I thought I should mention that v0.7 is getting flagged by virustotal. https://www.virustotal.com/gui/file/895349c623f2dafb3cb42902b10eefa92051f4fd6d750c0049c17cf2c30d294f/detection

**Comment by dslgr** [comment]
This did not seem to happen with v0.6 https://www.virustotal.com/gui/file/5875fe166aeaee97393e4383c121e97952136bfefe3db23ebe66d6db1422b5df

**Comment by ililim** [comment]
It's probably because I changed it from a "command line app" to an "invisible graphical app". I'm guessing these scanners find that suspicious. (This was done to improve the experience with launched from the new Windows Terminal).

The only thing that can be done AFAIK is to request that your antivirus vendor of choice investigates the false positive and updates their list. I've done so with Windows Defender as everyone has that one installed, but if someone uses a third party they will have to reach out to them.

**Comment by ulixxe** [comment]
I have the same issue with Symantec Endpoint Antivirus that found v0.7 infected.
This issue is not present in my [fork compilation](https://github.com/ulixxe/dual-key-remap-for-BigCtrl), check my [release](https://github.com/ulixxe/dual-key-remap-for-BigCtrl/releases/tag/BigCtrl).
I compiled it with Visual Studio 2022 on Windows 7.
Before compilation, I changed the code, but I think this issue doesn't depend on my code changes.
